deployments/gcp-uscentral1b/config/common.yaml

```diff
@@ -75,6 +80,8 @@ pangeo:
                 }
             },
           ]
+        prometheus: |
+          c.JupyterHub.authenticate_prometheus = False
```

Comment:
I think this makes it feasible for users within the kubernetes cluster to query the prometheus endpoints. I don't think we care about that, but I can adjust this if desired.